src/mcts/search.cc

```diff
@@ -366,6 +366,26 @@ std::pair<Move, Move> Search::GetBestMove() const {
   return GetBestMoveInternal();
 }
 
+bool Search::PopulateRootMoveLimit(MoveList* root_moves) const {
```

Comment:
Could you add a comment what this function returns?
And also why, it doesn't seem to be used anywhere. :)

Reply:
Its already commented in the header - and its used in the SearchWorker path to know whether to increment tb hit or not.